### DISCUSSION TITLE
Can we add a new configuration key?

### DISCUSSION THREAD

**Discussion by Tamir-zzz** [danielpinto8zz6/c-cpp-compile-run#338] [Ideas]
Can add a new c-cpp-compile-run.output-location-custom,
Custom output name for the compiled file, for example, output/a.exe
I don't like to output a new exe file every time the file is compiled, OCD is not suitable.
thx!!! : )

**Comment by danielpinto8zz6** [comment]
Hi, 
So, you mean the ability to change the executable name right? 
At the moment the name is generated based on the compiled file name, but yes, it can be added in a future release

**Reply by Tamir-zzz** [reply]
right!!! 
respect!

**Comment by L-Macy** [comment]
When I press F5, there is no response, the plug-in cannot be used, and there is no prompt.

**Comment by L-Macy** [comment]
Uploading Desktop 2023.11.06 - 17.54.23.01.mp4…
